components/arena/docker/arena_launcher.py

```diff
@@ -250,6 +250,9 @@ def main(argv=None):
   parser.add_argument('--timeout-hours', type=int,
                       default=200,
                       help='Time in hours to wait for the Job submitted by arena to complete')
+  parser.add_argument('--pending-timeout-minutes', type=int,
+                      default=360,
+                      help='Time in hours to wait for the Job submitted by arena from pending to running')
```

Comment:
The var says --pending-timeout-minutes, whereas the descriptions says hours?

Reply:
Thank you for spotting it！ Fix it.